tests/test_dynamic_sequence_scope.py

```diff
@@ -212,3 +212,45 @@ def test_dynamic_sequence_example():
     matches, features = capa.capabilities.dynamic.find_dynamic_capabilities(ruleset, extractor, disable_progress=True)
     assert r.name in matches
     assert 14 in get_call_ids(matches[r.name])
+
+
+# show how sequences that overlap a single event are handled.
+# TODO(williballenthin): but I think we really just want one match for this, not copies of the same thing.
```

Comment:
approaches:
- dedup these after matching (easy? slower)
- while matching advance window on match (nasty edge cases?)

Reply:
My current idea is: only emit a match if the prior address did not also have a match for that rule. This way we'll see where a region of behavior began, without having matches at every iteration of a tight loop (for example).

Reply:
see these changes: https://github.com/mandiant/capa/pull/2532/commits/b10d591857ba16d5cace4265cc507ce8be1f2a50

Reply:
great, thanks!